drivers/Makefile.include

```diff
@@ -103,3 +103,6 @@ endif
 ifneq (,$(filter veml6070,$(USEMODULE)))
     USEMODULE_INCLUDES += $(RIOTBASE)/drivers/veml6070/include
 endif
+ifneq (,$(filter uart_half_duplex,$(USEMODULE)))
```

Comment:
Well, I spoke too fast. The header file should go in `drivers/include/uart_half_duplex.h` and then this change in `Makefile.include` is not required anymore.